openQA.spec

```diff
@@ -229,6 +229,36 @@ rm -f t/00-tidy.t
 #make test
 rm -rf %{buildroot}/DB
 export LC_ALL=en_US.UTF-8
+# Skip tests not working currently, and Selenium tests
+for test in \
+    25-cache-service.t \
+    ui/01-list.t \
+    ui/02-list-group.t \
+    ui/06-operator_links.t \
+    ui/10-tests_overview.t \
+    ui/12-needle-edit.t \
+    ui/13-admin-no-login.t \
+    ui/13-admin.t \
+    ui/14-dashboard-parents.t \
+    ui/14-dashboard.t \
+    ui/15-admin-workers.t \
+    ui/15-comments.t \
+    ui/16-tests_dependencies.t \
+    ui/16-tests_job_next_previous.t \
+    ui/17-product-log.t \
+    ui/18-tests-details.t \
+    ui/19-tests-links.t \
+    ui/21-admin-needles.t \
+    ui/23-audit-log.t \
+    ui/24-feature-tour.t \
+    ui/25-developer_mode.t \
+    ui/26-jobs_restart.t
+do
+  echo "Removing t/$test"
```

Comment:
I don't think you need this line as every executed line should be echo'ed anyway